conf/messages

```diff
@@ -195,7 +195,7 @@ view_progress_formatter.we_received_your_return = We received your return and ar
 view_progress_formatter.amount_you_claimed = Amount you claimed
 view_progress_formatter.repayment_being_processed = Your repayment is being processed
 view_progress_formatter.estimated_repayment_date_passed = Estimated repayment date has passed
-view_progress_formatter.do_not_need_anything = You do not need to do anything right now. We are working on repaying you as soon as possible. </p><p class="govuk-body">We have sent you a letter to explain that your repayment is delayed.
+view_progress_formatter.do_not_need_anything = You do not need to do anything right now. We are working on repaying you as soon as possible. </p><p class="govuk-body">We''ll contact you with any further information about the delay.
```

Comment:
New content doesn't need own paragraph as per ticket requirements. The new sentence should just be added to end of the first paragraph and the second paragraph entirely removed.

Welsh & tests will need updating too.